src/components/stamp/stamp.js

```diff
@@ -13,8 +13,61 @@ export const availableTones = [
   'secondary',
 ];
 
+const getToneStyles = props => {
+  switch (props.tone) {
+    case 'critical': {
+      return css`
+        background-color: ${vars['--color-red-95']};
+        border: 1px solid ${vars['--color-red']};
+      `;
+    }
+    case 'warning': {
+      return css`
+        background-color: ${vars['--color-orange-95']};
+        border: 1px solid ${vars['--color-orange']};
+      `;
+    }
+    case 'positive': {
+      return css`
+        background-color: ${vars['--color-green-85']};
+        border: 1px solid ${vars['--color-green-40']};
+      `;
+    }
+    case 'information': {
+      return css`
+        background-color: ${vars['--color-blue-95']};
+        border: 1px solid ${vars['--color-blue']};
+      `;
+    }
+    case 'primary': {
+      return css`
+        background-color: ${vars['--color-green-95']};
+        border: 1px solid ${vars['--color-green-25']};
+      `;
+    }
+    case 'secondary': {
+      return css`
+        background-color: ${vars['--color-gray-90']};
+        border: 1px solid ${vars['--color-gray-60']};
+      `;
+    }
+    default:
+      return css``;
+  }
+};
+
 const Label = props => (
-  <div className={classnames(styles.container, styles[`tone-${props.tone}`])}>
+  <div
+    css={[
```

Comment:
Do you want to do it like this now instead of concatenating strings?

Reply:
I kind of like the array form for composition more to be fair. It feels a bit less bulky than template literals.

Reply:
Sure, I just need to know so that I can also follow that approach 😇

I just saw this now --> https://emotion.sh/docs/composition 😅

Reply:
Yeah it's kind of cool what they all offer also in relation to object styles. I guess we can just try out some approaches and then decide on one. Should all be internal to the library.